src/OrchardCore.Modules/OrchardCore.OpenId/Migrations/DefaultScopesMigration.cs

```diff
@@ -19,6 +19,18 @@ public int Create()
             var S = shellScope.ServiceProvider.GetService<IStringLocalizer<DefaultScopesMigration>>();
             var scopeManager = shellScope.ServiceProvider.GetRequiredService<IOpenIdScopeManager>();
 
+            if (await scopeManager.FindByNameAsync(OpenIddictConstants.Scopes.Email) == null)
+            {
+                var descriptor = new OpenIdScopeDescriptor
+                {
+                    DisplayName = S["Email"],
```

Comment:
Hum, I missed in the previous that that we're localizing the display name and the description. It's not wrong, but these fields will be populated using the locale used by the user when setting up the OpenID module, which may lead to unwanted results.

Note: OpenIddict has dedicated `LocalizedDisplayName` and `LocalizedDescriptions` properties that allow storing multiple localized versions (it's a dictionary using `CultureInfo` as the key).

Reply:
removed localization